pieceOfCake5_2.py

```diff
@@ -0,0 +1,10 @@
+def get_recipe_price(prices,optionals=None,**ingredients):
+  pricesCopy=copy.deepcopy(prices)
```

Comment:
Please use snake case when you write variable names (like you used it for get_recipe_price).